paper/src/main/java/vg/civcraft/mc/civmodcore/ACivMod.java

```diff
@@ -19,13 +19,25 @@
 import org.bukkit.event.Listener;
 import org.bukkit.event.server.PluginDisableEvent;
 import org.bukkit.plugin.Plugin;
+import org.bukkit.plugin.PluginDescriptionFile;
 import org.bukkit.plugin.java.JavaPlugin;
+import org.bukkit.plugin.java.JavaPluginLoader;
 import org.jetbrains.annotations.Contract;
 
 public abstract class ACivMod extends JavaPlugin {
 
 	private final Set<Class<? extends ConfigurationSerializable>> configClasses = new HashSet<>(0);
 
+	/** Primary constructor used by the real server */
+	protected ACivMod() {
```

Comment:
I made it protected here since `ACivMod` is abstract and so by definition the constructor can't be invoked publicly. Making abstract constructs protected is just standard practice, at least it has been in my experience. However I can make it public too if you'd prefer that.

Regardless, this change won't impact any extending classes since the sub-class constructor needs to be declared as public anyway.